composer.json

```diff
@@ -17,7 +17,7 @@
     "drupal/guzzle_cache": "^2.0",
     "symfony/cache": "^3.4",
     "highwire/drupal-psr-16": "^1.0",
-    "lullabot/mpx-php": "^0.8.2",
+    "lullabot/mpx-php": "^0.9.0",
```

Comment:
@m4olivei see https://github.com/nbc-bravo/d8/pull/1108#discussion_r241427119

Reply:
Did you mean this for @deviantintegral ?

Reply:
yep, but we already figured it out. Thanks @m4olivei!